.rubocop.yml

```diff
@@ -232,3 +232,7 @@ RSpec/SubjectStub:
   Enabled: false
 RSpec/LeadingSubject:
   Enabled: false
+RSpec/ContextWording:
+  Enabled: false
+RSpec/NestedGroups:
+  Enabled: false
```

Comment:
The Rails Lab voted to disable this only for MAIN. Can you update this PR, merge it, and we will create another one for MAIN?
```suggestion
```